apps/montreal_elixir/lib/montreal_elixir/social_feeds/meetup_event.ex

```diff
@@ -0,0 +1,6 @@
+defmodule MontrealElixir.SocialFeeds.MeetupEvent do
```

Comment:
Since `SocialFeeds` could eventually be used some something else (like videos), maybe this module be namespaced: `MontrealElixir.SocialFeeds.Meetup.Event`, and the file in folder `meetup`.